nodes/containers/nodes-containers-using.adoc

```diff
@@ -25,6 +25,22 @@ for years. {product-title} and
 Kubernetes add the ability to orchestrate containers across
 multi-host installations.
 
+[discrete]
+[id="nodes-containers-memory"]
+== About containers and RHEL kernel memory
+
+Due to Red Hat Enterprise Linux (RHEL) behavior, a container on a node with high CPU usage might seem to consume more memory than expected. The higher memory consumption could be caused by the kmem_cache in the RHEL kernel. The RHEL kernel creates a kmem_cache for each cgroup. For added performance, the kmem_cache contains a cpu_cache, and a node cache for any NUMA nodes. These caches all consume kernel memory. 
```

Comment:
Should `kmem_cache` and `cpu_cache` be marked up monospace throughout, being objects in the code?